hotel-management/src/hooks/room/useFetchRooms.ts

```diff
@@ -1,41 +1,43 @@
 import { useEffect, useState } from 'react';
 
-import { API_ENDPOINT } from '@/constants';
+import { API_ENDPOINT, ROOM_MESSAGE } from '@/constants';
 import HttpService from '@/services/httpService';
-import { Room, ApiResponse } from '@/types';
+import { Room, ApiResponse, RoomParams } from '@/types';
 
 export function useFetchRooms() {
```

Comment:
This hook should received params, so that when params has change, this hook has re-trigger the fetching inside